tron/bin/check_tron_jobs.py

```diff
@@ -293,41 +317,95 @@ def guess_realert_every(job):
     return realert_every
 
 
-def compute_check_result_for_job(client, job):
-    kwargs = {
-        "name": "check_tron_job.{}".format(job['name']),
-        "source": "tron",
+def sort_runs_by_interval(job_content, interval='day', until=None):
```

Comment:
I think I miss something here, but I can't find any place that until is not None.

Reply:
On line 333, I check `until`. If it is set by the caller, then I do nothing. But if its still `None`, I set it to the current time, so we create buckets up until the current time.

Reply:
I mean... is there any caller that sets until as not None?